tensorboard/webapp/metrics/views/card_renderer/image_card_directive.ts

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Directive, ElementRef} from '@angular/core';
+import {Store} from '@ngrx/store';
+
+import {State} from '../../../app_state';
+import {FEATURE_FLAGS_QUERY_STRING_NAME} from '../../../feature_flag/http/const';
+import {getFeatureFlagsToSendToServer} from '../../../feature_flag/store/feature_flag_selectors';
+
+@Directive({selector: 'img'})
```

Comment:
It's possible to merge these into a single directive: https://angular.io/api/core/Directive#options seems to say that you could have, for example, `selector: 'img, a'`.

I would mainly encourage merging them so that you don't have to write two sets of tests for these but otherwise the merging might not be worthwhile. I leave it up to you to decide whether merging is worthwhile.

Reply:
I wasn't able to cleanly merge them, but I added them to a common module so its a bit easier to use them.

Reply:
Merged into a single directive, thanks for the pointers.